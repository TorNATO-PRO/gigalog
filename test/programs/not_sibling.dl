parent(bob,alice).
parent(bob,charlie).

nonsibling(X,Y) :- parent(P,X), parent(P,Y), not same(X,Y).

same(X,X).
