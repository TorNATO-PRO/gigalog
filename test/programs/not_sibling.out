nonsibling(alice,charlie)
nonsibling(charlie,alice)
parent(bob,alice)
parent(bob,charlie)
same(alice,alice)
same(bob,bob)
same(charlie,charlie)
